src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -2773,14 +2773,12 @@ internal void DoCommand(EventCommandEventArgs commandArgs)
                 }
                 else
                 {
-#if !FEATURE_PERFTRACING
                     if (commandArgs.Command == EventCommand.SendManifest)
                     {
                         // TODO: should we generate the manifest here if we hadn't already?
                         if (m_rawManifest != null)
                             SendManifest(m_rawManifest);
                     }
-#endif
```

Comment:
Is there a test we can write for this?

Reply:
Also, did you audit the other uses of FEATURE_PERFTRACING to make sure the same issue doesn't exist elsewhere?

Reply:
Thanks Stephen - I was initially going to add the test to dotnet/diagnostics repo but figured that it's probably better to add some basic validation tests so I added a couple of tests to this PR. 

And yes, I went through every other usage of FEATURE_PERFTRACING and confirmed that we don't have a similar issue.